plugins/main/ptp/ptp.go

```diff
@@ -120,26 +127,33 @@ func setupContainerVeth(netns, ifName string, mtu int, pr *types.Result) (string
 	return hostVethName, err
 }
 
-func setupHostVeth(vethName string, ipConf *types.IPConfig) error {
+func setupHostVeth(vethName string, ipamResult *types.Result) error {
 	// hostVeth moved namespaces and may have a new ifindex
 	veth, err := netlink.LinkByName(vethName)
 	if err != nil {
 		return fmt.Errorf("failed to lookup %q: %v", vethName, err)
 	}
 
-	// TODO(eyakubovich): IPv6
+	ipConfig := ipamResult.IP6
+	cidrOnes := 128
+	cidrBits := 128
+	if ipConfig == nil {
```

Comment:
This behavior will prefer IPv6 over IPv4. Ideally we'd support both simultaneously, but we have discussed in other issues that it probably requires a spec revision. But can be done in this PR already is document the new behavior and then follow-up with a spec change.